usage/Installation.md

```diff
@@ -1,51 +1,56 @@
-This tutorial shows how to install the chat overflow framework. By now, working with the development environment is the only way. Currently, there are **no builds available.**
+## Installation
+This tutorial shows how to install the chat overflow framework. For now, working with the development environment is the only way. Currently, there are **no builds available.**
 
-### Pro
+<!-- ### Pro
 
 1. Clone the main repository. Then clone the api into folder `codeoverflow/api`, the gui into folder `codeoverflow/gui` and public plugins repository into `codeoverflow/plugins-public`.
 2. Setup the main repository as imported scala project, e.g. using IntelliJ. Make sure to refresh all sbt content and load the custom run configurations.
-3. You're done. Happy coding!
+3. You're done. Happy coding! -->
```

Comment:
Why have you commented out the Pro section?

Reply:
I think it could be confusing for users who think that they know sbt, scala, etc. and then meet the rather complex structure of the project. 

We can add that section back, no problem. But then I would rename it to something like "Setup for advanced users" or something like that.